htdocs/config/v2/blis_cloud_server.php

```diff
@@ -5,11 +5,12 @@
 require_once(__DIR__."/lib/analyzed_backup.php");
 require_once(__DIR__."/lib/backup.php");
 require_once(__DIR__."/lib/encrypted_file.php");
+require_once(__DIR__."/../../includes/db_lib.php");
 
 # The controller for BLIS Cloud Server (/receiver) operations.
 
 if (!Features::allow_client_connections()) {
-    header('HTTP/1.1 404 Not Found', true, 404);
```

Comment:
This made me take a look at the HTTP protocol docs since I wasn't sure if the reason message ("Not found") can actually be replaced. It turns out... [it can!](https://httpwg.org/specs/rfc9110.html#rfc.section.15.1) That said, I wouldn't recommend taking the time to localize these going forward since they are not (usually) user-facing messages.